e2e/playwright/editor-tests.spec.ts

```diff
@@ -1274,4 +1273,60 @@ sketch001 = startSketchOn(XZ)
       await middleMousePan(800, 200, 900, 300)
     })
   })
+
+  test(`Only show axis planes when there are no errors`, async ({
+    page,
+    homePage,
+  }, testInfo) => {
+    await page.addInitScript(async () => {
+      // extrude length = 0 causes an error
+      localStorage.setItem(
+        'persistCode',
+        `sketch001 = startSketchOn(XZ)
+    profile001 = circle(sketch001, center = [-100.0, -100.0], radius = 50.0)
+
+    sketch002 = startSketchOn(XZ)
+    profile002 = circle(sketch002, center = [-100.0, 100.0], radius = 50.0)
+    extrude001 = extrude(profile002, length = 0)` // length = 0 is causing an error
+      )
+    })
+
+    const viewportSize = { width: 1200, height: 800 }
+    await page.setBodyDimensions(viewportSize)
+
+    await homePage.goToModelingScene()
+
+    const u = await getUtils(page)
+    const locationToHaveColor = async (
+      position: { x: number; y: number },
+      color: [number, number, number]
+    ) => {
+      return u.getGreatestPixDiff(position, color)
+    }
+
+    await u.waitForPageLoad()
+
+    // Wait until axis planes are rendered.
+    await u.openDebugPanel()
+    await u.expectCmdLog('[data-message-type="execution-done"]')
+    await u.closeDebugPanel()
+
+    // Alternatively just wait a bit
+    // await page.waitForTimeout(3000)
+
+    await expect
+      .poll(
+        async () =>
+          locationToHaveColor(
+            // This is a position where the blue part of the axis plane is visible if its rendered
+            { x: viewportSize.width * 0.75, y: viewportSize.height * 0.2 },
+            TEST_COLORS.DARK_MODE_BKGD
+          ),
+        {
+          timeout: 5000,
+          message: 'XZ plane color is visible',
+        }
+      )
+      .toBeLessThan(15)
```

Comment:
There's a util already for this.
```suggestion
      await scene.expectPixelColor(
        TEST_COLORS.DARK_MODE_BKGD,
        // This is a position where the blue part of the axis plane is visible if its rendered
        { x: viewportSize.width * 0.75, y: viewportSize.height * 0.2 },
        15
      )
```

Reply:
thanks, updated it!